android/src/main/res/values/strings.xml

```diff
@@ -1,19 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
Can we remove this file ?

Reply:
We can remove this file, but we have to set the string in a Java or Kotlin file. It is better to save this file because it is best practice to declare string variables.